floris/tools/floris_interface.py

```diff
@@ -267,15 +267,18 @@ def _reinitialize(
                 (wind_directions is not None)
                 or (wind_speeds is not None)
                 or (turbulence_intensities is not None)
+                or (heterogenous_inflow_config is not None)
```

Comment:
I also noticed "heterogenous" is misspelled throughout the code. Although not necessarily related to this PR, it would be good to fix it at some point.

Reply:
There are many `time_series` flags scattered throughout also. Since neither of these two things fall into another category, I can submit a pull request to fix these two together.